Source/Renderer/createUniform.js

```diff
@@ -167,7 +169,9 @@ define([
                 this._gl.uniform4f(this._location, v.x, v.y, v.z, v.w);
             }
         } else {
+            //>>includeStart('debug', pragmas.debug);
```

Comment:
Keeping the throw statement inside the else block keeps it simple, we don't have to write a separate if statement and repeat all previous conditions. However, is there a cost to having empty else blocks after build optimization? Or are empty branches actually cut out? @mramato